CONTRIBUTING.md

```diff
@@ -148,14 +148,30 @@ During a peer review, reviewers will add a tag to the comment to help the author
 [Kudos]|The reviewer liked this, specifically. Go you.
 [Suggestion]| An optional, alternative approach.
 
-
-
 Adding multiple tags to a comment is fine.  Do not combine tags.
 
 Good: ``[Suggestion][Kudos] This is great, but...``
 
 Bad:  ``[Suggestion|Kudos] This is great, but...``
 
+
+## Pull request labels
+Labels may be applied to your pull requests, for the purposes of organization and / or overall 
+status.
+
+| label | Explanation |
+| --- | --- |
+Awaiting changes    | this pull request is on hold until requested changes have been effected
```

Comment:
[Discussion] Should we maybe have an 'Awaiting response' label as well?
There are several ways that a PR can be stalled on the submitter that doesn't necessarily involve code changes.
Alternatively, 'Awaiting response' could replace 'Awaiting changes' entirely, and simplify the number of labels..but it's a little less clear what action is required if we only use one label for both.

Reply:
I feel `awaiting response` would be added and removed too frequently and is therefore too broad.